packages/wallet/ui/src/components/AppBar.jsx

```diff
@@ -47,9 +64,47 @@ const useStyles = makeStyles(theme => ({
 }));
 
 // Exported for testing only.
-export const AppBarWithoutContext = ({ useChainBackend }) => {
+export const AppBarWithoutContext = ({
+  connectionComponent,
+  connectionState,
+  wantConnection,
+  setWantConnection,
+}) => {
   const theme = useTheme();
   const classes = useStyles(theme);
+  const [status, setStatus] = useState(
+    wantConnection ? 'connecting' : 'disconnected',
```

Comment:
please define an enum-like object for these strings, to make explicit what the domain of `status` is